core/Dockerfile.template

```diff
@@ -16,7 +16,7 @@ ENV DBUS_SYSTEM_BUS_ADDRESS=unix:path=/host/run/dbus/system_bus_socket
 
 ENV npm_config_unsafe_perm=true
 
-RUN install_packages jq git vim rsync unzip bluez
+RUN install_packages jq git vim rsync unzip bluez qemu-utils
```

Comment:
So is one of the failures missing `qemu-img` which is used during the flash to expand the image? If this is the case, we should really handle this error instead of silently ignoring it.

Reply:
`qemu-img` is required by https://github.com/balena-os/balena-image-flasher-unwrap

Reply:
We shouldn't need this with the new node unwrap approach, qemu-img is only required for some optional bits of the external unwrap script.